.gitignore

```diff
@@ -27,3 +27,4 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+package-lock.json
```

Comment:
please use yarn for installation of dependencies. There is already a yarn.lock file.